src/modules/Builder/components/Configuration/components/Configuration/Configuration.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+
+import Main from "../Main/Main";
+import Wireless from "../Wireless/Wireless";
+
+import { fetchConfigurationParams } from "../../actions/configuration";
+import { wirelessSupportOS, wirelessSupportDevices } from "./config";
```

Comment:
Define `wirelessSupportOS` and `wirelessSupportDevices` in this module and remove `config.js`.